1929.concatenation-of-array.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/concatenation-of-array/
+*/
+
+var getConcatenation = function(nums) {
+    let ans = nums.concat(nums);
```

Comment:
use const

Reply:
but it is better i guess.
```suggestion
    return nums.concat(nums);
```

Reply:
I modified the code by using the return statement as your suggestion.